modDesc.xml

```diff
@@ -5,7 +5,7 @@
     <title>
         <en>Target Fill Level</en>
         <de>Ziel-Füllstand</de>
-        <ru>Уровень Заполнения Транспорта</ru>
+        <ru>Уровень Заполнения ТС И Хранилищ</ru>
```

Comment:
As a non russian speaker: what does TC stand for and is this a well known abbreviation?